test/test-client.js

```diff
@@ -1,6 +1,6 @@
 /*global test, asyncTest, $, start,
   setFeatures, ok, deepEqual, equal*/
-'use strict';
```

Comment:
@ericelliott Sorry, but I need to use `with` statement to mock window object. Because if I use window.location then page is redirected and ironically qunit also crash on invalid GET parameters 😄